aspnetcore/fundamentals/servers/yarp/yarp-overview.md

```diff
@@ -0,0 +1,66 @@
+  - Control Over URL-Space: Incoming URLs can be transformed before forwarding to the backend. This abstracts the external URLs from how they are mapped to internal services. Internal service endpoints can change without affecting external URLs.
+- Security: Internal service endpoints can be hidden from external exposure, protecting against some types of cyber attacks such as [DDoS attacks](https://www.microsoft.com/security/business/security-101/what-is-a-ddos-attack?msockid=3e35ed3aa4666d8003aaf830a5006c74) and [SQL injection ](/sql/relational-databases/security/sql-injection).
+- Caching: Frequently requested resources can be cached to reduce the load on backend servers and improve response times.
+- Versioning: Different versions of an API can be supported using different URL mappings.
+
+## Why use a reverse proxy
+
+Using a reverse proxy offers several advantages:
+
+- Scalability: By distributing traffic across multiple servers, a reverse proxy helps scale apps to handle more users and higher loads. Backend servers scaled (added or removed) without impacting the client.
+- Improved performance: Caching and load balancing can significantly enhance the performance, latency, and responsiveness of apps.
+- Enhanced security: A reverse proxy can provide an additional layer of security by hiding backend servers and mitigating certain types of attacks such as [DDoS attacks](https://www.microsoft.com/security/business/security-101/what-is-a-ddos-attack?msockid=3e35ed3aa4666d8003aaf830a5006c74) and [SQL injection ](/sql/relational-databases/security/sql-injection). YARP can simplify authentication and authorization.
+- Simplified maintenance: Reverse proxies can handle [SSL/TSL Termination](/azure/application-gateway/ssl-overview) and other tasks, simplifying the configuration and maintenance of backend servers. For example, SSL certificates and security policies can be managed at the reverse proxy level instead of on each individual server.
+
+## How a reverse proxy handles HTTP
+
+A reverse proxy handles HTTP requests and responses in the following manner:
+
+- Receiving  requests: The reverse proxy listens on specified ports and endpoints for incoming HTTP requests from clients.
+- Terminating Connections: The inbound HTTP connections are terminated at the proxy and new connections are used for outbound requests to destinations.
+- Routing  requests: Based on predefined routing rules and configurations, the reverse proxy determines which backend server, or cluster of servers, should handle the request.
+- Forwarding  requests: The reverse proxy forwards the client request to the appropriate backend server, transforming the path and headers as necessary.
+- Connection pooling: The outbound connections are pooled to reduce connection overhead and make most use of HTTP 1.1 pipelining and parallel requests with HTTP/2 and HTTP/3.
```

Comment:
```suggestion
- Connection pooling: The outbound connections are pooled to reduce connection overhead and make most use of HTTP 1.1 reuse and parallel requests with HTTP/2 and HTTP/3.
```
HTTP/1.1 pipelining is a feature we're intentionally not supporting at the HttpClient layer.